packages/components/pie-button/README.md

```diff
@@ -86,6 +86,7 @@ import { PieButton } from '@justeattakeaway/pie-button/dist/react';
 | disabled       | `Boolean` | `false`    | If `true`, disables the button.                                                                                      |
 | isFullWidth    | `Boolean` | `false`    | If `true`, sets the button width to 100% of its container.                                                          |
 | isLoading      | `Boolean` | `false`    | If `true`, displays a loading indicator inside the button.                                                           |
+| isResponsive   | `Boolean` | `false`    | If `true`, uses the next larger size on wide viewports.                                                              |
```

Comment:
I know the button docs PR went in before this PR, but we need to update the `props.json` file in the docs to reflect this new prop until we entirely remove the props table from the readme